realm/realm-library/src/androidTest/java/io/realm/LinkingObjectsQueryTests.java

```diff
@@ -401,11 +400,11 @@ public void isEmpty_acrossLink() {
                 case LINKING_OBJECTS:
                     // Rows 0 and 1 are not included as they are linked to another row through FIELD_OBJECT
                     // Row 2 is included (no link)
-                    assertEquals(1, realm.where(AllJavaTypesUnsupportedTypes.class).isEmpty(AllJavaTypesUnsupportedTypes.FIELD_OBJECT + "." + AllJavaTypesUnsupportedTypes.FIELD_LO_OBJECT).count());
```

Comment:
Why the expected values changed for these tests?

Reply:
It was changed after https://github.com/realm/realm-core/pull/6867. There was a lot of discussion about this, but in the end, we determined that the old behavior was buggy and subsequently fixed.